### PR TITLE
Cleaned up a bunch of CA counties; deprecated some, fixed others.

### PR DESCRIPTION
**Pull request by chunder**

### CODE REVIEW

File: src/shared/scrapers/USA/CA/orange-county.js [hunk 1]
```diff
@@ -11,20 +11,26 @@ const scraper = {
...
+        ),
+        deaths: parse.number(
+          $('td:contains("Total Deaths")')
+            .next()
+            .text()
+        )
+      };
+    },
+    '2020-3-18': async function scraper() {
+      // https://occovid19.ochealthinfo.com/coronavirus-in-oc
```
Comment (lazd) [inline src/shared/scrapers/USA/CA/orange-county.js]:
```suggestion
      this.url = 'https://occovid19.ochealthinfo.com/coronavirus-in-oc';
      await fetch.page(this.url);
```

File: src/shared/scrapers/USA/CA/san-benito-county.js [hunk 2]
```diff
@@ -11,31 +11,36 @@ const scraper = {
...
+        recovered: parse.number(
+          $table
+            .find('td:contains("Recovered")')
+            .next('td')
+            .text()
+        )
+      };
+    },
+    '2020-3-19': async function scraper() {
+      throw new Error('Need to scrape new arcgis');
```
Comment (lazd) [inline src/shared/scrapers/USA/CA/san-benito-county.js]: Pull down the CSV if you have a URL so we can have it in cache for timeseries purposes.

**Comment by chunder** [comment]
It's no good to scrape the page any more. All the data is inside arcgis. And I'm not going to spend the time figuring that out right now.

**Comment by cmcjacob** [comment]
> It's no good to scrape the page any more. All the data is inside arcgis. And I'm not going to spend the time figuring that out right now.

Can you elaborate?  If the GIS data is coming from state resources, i can't see how it could be "no good"

**Comment by chunder** [comment]
Scraping the gis data would be great. But it takes work to figure out the right url to scrape. Until someone puts in that work, it’s not worth scraping the old url with no data.

**Pull request merged**